test/pass/test_binary.sest

```diff
@@ -0,0 +1,20 @@
+module TestBinary = struct
```

Comment:
Nice tests! It would be better if this file has additional tests about strings that include `"`.

Reply:
Updated!